sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/rule/EncryptRule.java

```diff
@@ -132,7 +132,17 @@ public String getLogicColumnOfCipher(final String logicTable, final String ciphe
      * @return plain column
      */
     public Optional<String> findPlainColumn(final String logicTable, final String logicColumn) {
-        return tables.containsKey(logicTable) ? tables.get(logicTable).findPlainColumn(logicColumn) : Optional.<String>absent();
+        Optional<String> originColumnName = getOriginColumnName(logicTable, logicColumn);
+        return tables.containsKey(logicTable) && originColumnName.isPresent() ? tables.get(logicTable).findPlainColumn(originColumnName.get()) : Optional.<String>absent();
```

Comment:
Can you move `originColumnName.isPresent()` before `tables.containsKey(logicTable)`?

Reason:
1. Let's keep consistent with `EncryptTable.java`
2. `originColumnName.isPresent()` should judge first , if absent, the statement should circuit break, do not execute `tables.containsKey(logicTable)`

Reply:
Accept, had committed~